build.toml

```diff
@@ -52,3 +60,4 @@ package_output_dir = 'Built'
 [package.payload_map]
 'Built\\Scan Engine' = 'documents\National Instruments\NI VeriStand {veristand_version}\Custom Devices\Scan Engine'
 'Built\\Errors' = 'ni-paths-NISHAREDDIR\LabVIEW Run-Time\{veristand_version}\errors\English'
+'Built\\Scripting\\Scan Engine' = 'ni-paths-LV{veristand_version}DIR\vi.lib\addons\VeriStand Custom Device Scripting APIs\Scan Engine'
```

Comment:
This is currently adding a new set of files to the existing package. You will instead want to create a new package section (which will involve turning the first package listing into an element of a collection by adding an extra set of brackets - calling out explicitly because it's easy to miss).

See https://github.com/ni/niveristand-data-sharing-framework-custom-device/blob/bc04e6683cc8c5b0de0793d325aaddd9cd3469fa/build.toml#L120-L134 for an example.

Reply:
I quickly threw together the following, which seems like a good starting point. Untested, but probably close to correct.

```toml
[[package]]
type = 'nipkg'
package_output_dir = 'Built'

[package.payload_map]
'Built\\Scan Engine' = 'documents\National Instruments\NI VeriStand {veristand_version}\Custom Devices\Scan Engine'
'Built\\Errors' = 'ni-paths-NISHAREDDIR\LabVIEW Run-Time\{veristand_version}\errors\English'

[[package]]
type = 'nipkg'
control_file = 'control_scripting'
package_output_dir = 'Built'

[package.payload_map]
'Built\\Scripting\\Scan Engine' = 'ni-paths-LV{veristand_version}DIR\vi.lib\addons\VeriStand Custom Device Scripting APIs\Scan Engine' 
```